src/g.cpp

```diff
@@ -460,39 +478,45 @@ optional<g1> g1::multiExp(std::span<const g1> points, std::span<const array<uint
     {
         for(uint64_t i = 0; i < bucketSize; i++)
         {
+            if (i & 255 == 0) {
```

Comment:
We should probably check that yield is not null:
```suggestion
            if (i % 256 == 0 && yield) {
```

Reply:
change to if (yield && ((i & 255) == 0))